### ISSUE TITLE
[Feature Request] Tachiyomi Extension

### ISSUE THREAD

**Issue by Septolum** [arsonistAnt/Wyrmprint#8]
Have you considered converting this into a Tachiyomi Extension?

https://github.com/inorichi/tachiyomi

**Comment by arsonistAnt** [comment]
Yes, I have! Was going to actually work on doing that after getting this app fleshed out.

**Comment by Septolum** [comment]
Great! I look forward to it :smile:

**Comment by Adoniweeb** [comment]
Can u please add Shield manga it has a lot of good manga that some of the extensions don't have on the app but I love the app it's amazing tnx
https://shieldmanga.club/
